### DISCUSSION TITLE
How can I make a button persistent?

### DISCUSSION THREAD

**Discussion by OldPranoy** [Breq16/flask-discord-interactions#125] [Q&A]
Im making an button role system, so after a restart buttons are showing interaction failed so is there a way to make it persistent!

**Comment by breqdev** [comment]
Take a look at the documentation on [Custom ID Internals](https://flask-discord-interactions.readthedocs.io/en/latest/components.html#custom-id-internals). In short, you likely want to explicitly pass a string to the "custom_id" parameter:

```python
@discord.custom_handler(custom_id="your_custom_interaction_id_goes_here")
def handle_click(ctx):
    # handler goes here
    return Message(...)
```

The actual string itself does not matter, but strings should be unique (two buttons should not have the same custom ID string).

Let me know if this solves your issue!
